packages/block-library/src/navigation/editor.scss

```diff
@@ -61,11 +61,17 @@
 	// Polish the Appender.
 	.wp-block-navigation .block-list-appender {
 		margin: 0;
+		width: 28px;
+		height: 28px;
+		margin-top: $grid-size/2;
 
 		.block-editor-button-block-appender {
 			padding: $grid-size;
+			padding-bottom: $grid-size/2;
+			padding-top: $grid-size/2;
 			outline: none;
 			background: none;
+			border-radius: 50%;
```

Comment:
This seems to be making the button block appender look like the default appender. Could the default appender be used instead? I'm a little confused because the change in `NavigationLinkEdit` above looks like it does that.

Reply:
I removed this and the visual kept OK, probably it is a leftover before I switched to the default appender or something.